Workflow/Sources/AnyWorkflowConvertible.swift

```diff
@@ -37,11 +37,15 @@ extension AnyWorkflowConvertible {
     /// - Parameter key: A string that uniquely identifies this workflow.
     ///
     /// - Returns: The `Rendering` generated by the workflow.
-    public func rendered<Parent>(with context: RenderContext<Parent>, key: String = "") -> Rendering where Output: WorkflowAction, Output.WorkflowType == Parent {
```

Comment:
Did we want to keep the old funcs around + deprecated to ease adoption?

Reply:
Good idea, added the `deprecated` versions in.